lib/WorldState.js

```diff
@@ -79,11 +78,12 @@ class WorldState {
    * @param {string} json The worldstate JSON string
    * @param {Object} [deps] The options object
    */
-  constructor(json, deps = defaultDeps) {
+  constructor(json, lang = 'en_US', deps = defaultDeps) {
```

Comment:
when you create each, you can either use a flag that says "this is the top level", or however you'd prefer to do that.